### ISSUE TITLE
Integrating Real Bus Data

### ISSUE THREAD

**Issue by PaulOOlabisi** [googlemaps/transport-tracker#52]
Hello @domesticmouse , I am following the google transport tracker tutorial as well, and I would like to use the real bus location data. I have the android app set up following [this ](https://developers.google.com/maps/solutions/transport-tracker/start#step_6_set_up_the_android_app), and I have changed this to `"simulation": false`. The part where I'm stuck is here;
`if (trackerConfig.simulation) {`
  `const {BusSimulator} = require('./bus_simulator.js');`
  `const generatedPaths = require('./paths.json');`
  `new BusSimulator(timeRef, gtfs, busLocationsRef, generatedPaths);`
`} else {`
  `// Exercise for the reader: integrate real bus location data`
`}`

**Comment by domesticmouse** [comment]
As mentioned elsewhere, please go through https://codelabs.developers.google.com/codelabs/realtime-asset-tracking/index.html

Thanks!

**Comment by domesticmouse** [comment]
No, we haven't had the resources to built official integrations for all of the various frameworks (and micro-frameworks) that litter the JavaScript scene. Angular usually has a way of exposing the content div at which point you can use the official Google Maps JS API to attach a map and manage it from there.

**Comment by AbeeraTariq02787** [comment]
> @domesticmouse I integrated the real bus location data and it's working, redesigned the front end but I would like to try and implement the transport tracker map code into an angular project. I'm new to angular, javascript and node js and having issues finding a google maps for angular. I've found some called @types/google-maps and angular-maps but the documentation is terrible and hard to follow. Is there an official Google Maps API with the Roads and Direction feature for Angluar?

Hi! I am new to android and web development and struggling to find the answer on the links referenced earlier. Can you please guide me which part of the code needs to inserted in the else clause?

Thank you!
